addons/editor-extra-keys/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Extra key support",
-  "description": "Adds more keys to the \"key () pressed?\" and \"when () key pressed\" block dropdowns, such as enter, dot, comma, and more.",
+  "description": "Adds more keys to the \"key () pressed?\" and \"when () key pressed\" block dropdowns, such as \"Enter\", period (full stop), comma, and more.",
```

Comment:
The letter is called "dot" in computing context, not "period" or "full stop", according to "en period wikipedia full stop com".

Reply:
Chrome calls the key (not the character) "period".
![image](https://user-images.githubusercontent.com/51849865/155684950-fdce5a4c-4f1b-441e-a605-fbbb0e16dbc6.png)